### PR TITLE
Add pod labels support to external-snapshotter

### PR DESCRIPTION
**Pull request by gohmc**
This PR allows adding labels to external snapshotter pod and external snapshotter validator pod.

### CODE REVIEW

File: charts/snapshot-controller/README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ The following options are available:
 | `image.tag`                 | Override the tag to pull. If not given, defaults to charts `AppVersion`.                                               | `""`                                                                                               |
 | `imagePullSecrets`          | Image pull secrets to add to the deployment.                                                                           | `[]`                                                                                               |
 | `podAnnotations`            | Annotations to add to every pod in the deployment.                                                                     | `{}`                                                                                               |
+| `podLabels`                 | Labels to add to every pod in the deployment.                                                                         | `{}`                                                                                               |
```
Comment (WanzenBug) [inline charts/snapshot-controller/README.md:93]: Please align the table separators

File: charts/snapshot-validation-webhook/README.md [hunk 2]
```diff
@@ -108,6 +108,7 @@ available.
 | `tls.certManagerIssuerRef`           | Issuer to use for provisioning the TLS certificate. If this is used, `tls.certificateSecret` can be left empty.        | `{}`                                                                                               |
 | `imagePullSecrets`                   | Image pull secrets to add to the deployment.                                                                           | `[]`                                                                                               |
 | `podAnnotations`                     | Annotations to add to every pod in the deployment.                                                                     | `{}`                                                                                               |
+| `podLabels`                          | Labels to add to every pod in the deployment.                                                                         | `{}`                                                                                               |
```
Comment (WanzenBug) [inline charts/snapshot-validation-webhook/README.md:111]: Same as above: please align the table separators

**Review by WanzenBug — CHANGES_REQUESTED** [review]
Overall, let's get this merged. Just a few small nits to solve.

Please make sure to follow the instructions for the DCO check: https://github.com/piraeusdatastore/helm-charts/pull/26/checks?check_run_id=15075928679

**Comment by tanvp112** [comment]
@WanzenBug, added the line "Signed-off-by" to the commit message and updated the table in read me.

**Review by WanzenBug — APPROVED** [review]
LGTM. I squashed both commits into one.

**Pull request merged**